openstates/md/bills.py

```diff
@@ -165,10 +170,132 @@ def parse_bill_votes(self, doc, bill):
                 vote.add_source(href)
                 yield vote
 
+    def parse_bill_votes_new(self, doc, bill):
+        elems = doc.xpath('//a')
+
+        # MD has a habit of listing votes twice
+        seen_votes = set()
+
+        for elem in elems:
+            href = elem.get('href')
+            if (href and "votes" in href and href.endswith('pdf') and
+                    ("Senate" in href or "House" in href) and href not in seen_votes):
```

Comment:
Here you check for `Senate` and `House`; later you assign chamber based on `senate` and `house`. Which capitalization test is correct?

Reply:
It looks like the uppercase version is what should be checked for; I'm seeing falsely-assigned `lower` when I spot-check the data (eg, for http://mgaleg.maryland.gov/2018RS/votes/Senate/0378.pdf).

Reply:
@mileswwatkins I didn't get this one.

Reply:
Got it Fixed!!